pkg/controller/core/localqueue_controller.go

```diff
@@ -128,7 +128,7 @@ func (r *LocalQueueReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 	if meta.IsStatusConditionTrue(cq.Status.Conditions, kueue.ClusterQueueActive) {
-		err = r.UpdateStatusIfChanged(ctx, &queueObj, metav1.ConditionTrue, "Ready", "Can submit new workloads to clusterQueue")
+		err = r.UpdateStatusIfChanged(ctx, &queueObj, metav1.ConditionTrue, "Ready", "Can submit new workloads to localQueue")
```

Comment:
Hm, wondering if we should cherry-pick this, because this is actually a user-facing change.

Ideally such changes are in a separate PR so that we can make a decision about cherry-pick independent of other changes. The cleanup changes are very small, and I feel confident about them, but would you mind splitting this out into independent pieces (cleanup and tiny fix) @PBundyra ?

Reply:
Sure, on it

Reply:
https://github.com/kubernetes-sigs/kueue/pull/4955